core/chains/evm/logpoller/orm.go

```diff
@@ -338,7 +338,7 @@ func (o *DbORM) SelectLogsByBlockRange(start, end int64) ([]Log, error) {
         	WHERE evm_chain_id = :evm_chain_id
         	AND block_number >= :start_block 
         	AND block_number <= :end_block 
-        	ORDER BY (block_number, log_index, created_at)`, args)
+        	ORDER BY (block_number, log_index)`, args)
```

Comment:
There should not be 2 logs with the same log_index, block_number and evm_chain_id